README.md

```diff
@@ -1,6 +1,42 @@
-# Keycloak-js
+# Keycloak JS
 
-Client-side JavaScript library that can be used to secure web applications and integrate them with Keycloak.
+Keycloak is an Open Source Identity and Access Management solution for modern Applications and Services.
 
-TODO
+This reporsitory contains the client-side JavaScript library that can be used to secure web applications and integrate them with Keycloak.
 
+## Help and Documentation
+
+* [Documentation](https://www.keycloak.org/documentation.html)
+* [Keycloak JavaScript adapter guide](https://www.keycloak.org/securing-apps/javascript-adapter) - The specific guide for this client
+* [User Mailing List](https://groups.google.com/g/keycloak-user) - Mailing list for help and general questions about Keycloak
+* [Issue Tracker](https://github.com/keycloak/keycloak-nodejs-connect/issues) - Issue tracker for bugs and feature requests
```

Comment:
This one needs to be updated to use this repository.

Reply:
Uups! I missed this one. Thanks!